### DISCUSSION TITLE
Viewing Image data

### DISCUSSION THREAD

**Discussion by Kolaparthi-Vamsi** [MIMBCD-UI/dataset-uta4-dicom#2] [Q&A] [answered]
I believe the datasets are image data. How to view them. I am from LTI organization and I am researching on Breast cancer classification as part of project. Please can you help me in viewing the images.

**Comment by FMCalisto** [comment]
Hello @Kolaparthi-Vamsi, thank you for your interest in this dataset. You can use our tools, such as the ones available in [`prototype-breast-screening`](https://github.com/MIMBCD-UI/prototype-breast-screening) and [`prototype-multi-modality`](https://github.com/MIMBCD-UI/prototype-multi-modality) repositories, or you can download a viewer. For instance, you can use the [Weasis DICOM viewer](https://nroduit.github.io) for your purpose. Hope that helps!

**Reply by Kolaparthi-Vamsi** [reply]
Thank you for the reply, I have a doubt. I am doing a capstone project with BITS Pilani education institution. The idea is to develop a image classification Deep Learning model and after developing it for a test image when sent to a model, the expected is to identify the degree based on the training and also develop a report for the image. So we need reports as well in our input and develop a model to perform NLG. 

Please my request, if you can help me with the image data with labels and the reports as well.

**Reply by Kolaparthi-Vamsi** [reply]
I am looking into this: https://github.com/MIMBCD-UI/dataset-uta4-dicom/commit/45d9faedbfbe9f7afbc1104809a04feb518db84f

I am seeing this images. But want to know what this data is about and how to labels. Please can you help. Do we have any reports related for this images.

**Comment by FMCalisto** [comment]
Your idea is a great effort, congratulations to you and to your team. It seems you are doing the right thing.

Unfortunately, we do not have any reports, nor information about it. However, you can look at the labels we have for this *dataset*. Just follow both [`dataset-uta4-rates`](https://github.com/MIMBCD-UI/dataset-uta4-rates) and [`dataset-uta4-classifications`](https://github.com/MIMBCD-UI/dataset-uta4-classifications) repositories. Any **[Related Repositories](https://github.com/MIMBCD-UI/dataset-uta4-dicom#related-repositories)** are available [here](https://github.com/MIMBCD-UI/dataset-uta4-dicom#related-repositories).

Please read the [`README.md`](https://github.com/MIMBCD-UI/dataset-uta4-dicom/blob/master/README.md) file. There is important information you can find on it.

**Reply by Kolaparthi-Vamsi** [reply]
Thank you Calisto. I am looking into this: https://github.com/MIMBCD-UI/dataset-uta4-dicom/commit/45d9faedbfbe9f7afbc1104809a04feb518db84f#diff-c104966a65e7140f1738883560240267ed86a1823a85ed761ffaf1fe43e2b0be

Can you help me here what is mm and sm folders. What do they mean. Also under each mm and sm folders there are sub folders, what do they mean. Please can you help me in understanding data.
My humble request.

**Reply by Kolaparthi-Vamsi** [reply]
Sorry Calisto, I understood what is SM and MM . Single modality and Multi modality. OK why do we have sub folders under each SM and MM, please can you help me with that.

**Comment by FMCalisto** [comment]
This *dataset* was used under our [Human-Computer Interaction](https://www.sciencedirect.com/topics/computer-science/human-computer-interaction) (HCI) research. Our research deals with several user studies, bridging together the various aspects of the HCI topics. In [this study](https://doi.org/10.1145/3399715.3399744) [1], we describe the field research, design and comparative deployment of a multimodal medical imaging user interface for breast screening. The goal was to do a comparison between Single-Modality (SM) and Multi-Modality (MM) screening of breast cancer diagnosis. For that, we divided the *dataset* to achieve the main goals of the study. You can find more about that in this [Master Thesis](http://dx.doi.org/10.13140/RG.2.2.15187.02084) [2] on **page 54** of the document.

[1] Francisco Maria Calisto, Nuno Nunes, and Jacinto C. Nascimento. 2020. BreastScreening: On the Use of Multi-Modality in Medical Imaging Diagnosis. In Proceedings of the International Conference on Advanced Visual Interfaces (AVI '20). Association for Computing Machinery, New York, NY, USA, Article 49, 1–5. https://doi.org/10.1145/3399715.3399744

[2] Calisto, F. M. (2017). Medical imaging multimodality breast cancer diagnosis user interface. Master's thesis. Instituto Superior Técnico, Avenida Rovisco Pais, 1. http://dx.doi.org/10.13140/RG.2.2.15187.02084

**Reply by Kolaparthi-Vamsi** [reply]
Thank you Calisto, your suggestion is really helpful and I am going with the info you provided. Really honored to get insights from you. Have a great day Calisto.

**Comment by Kolaparthi-Vamsi** [comment]
Thank you so much Calisto. I am very happy to get answers from you and be as part of your network. I am reading about what you have sent, doing some research. There are two labels to identify from the images that if it is a SM or MM. Is my understanding right.
If so, do we have images with classes relating to degree of the cancer riskiness or something like that. Please can you point me directly to the url here which has the relevant data. My humble request.

**Comment by FMCalisto** [comment]
Yes, you have the [`dataset-uta7-dicom`](https://github.com/MIMBCD-UI/dataset-uta7-dicom) repository, which will be even more complete. You just need to extract the [BIRADS](https://radiopaedia.org/articles/breast-imaging-reporting-and-data-system-bi-rads) from the [`anonymous_patient_id_birads_data_ready.csv`](https://github.com/MIMBCD-UI/dataset-uta7-rates/blob/master/dataset/anonymous_patient_id_birads_data_ready.csv) file.

**Reply by Kolaparthi-Vamsi** [reply]
Thank you a lot. Will look into it.

**Reply by Kolaparthi-Vamsi** [reply]
Calisto, Greetings. Hope you are doing great.
The whole total image classification problem here is experimented with images here: https://github.com/MIMBCD-UI/dataset-uta4-dicom/commit/45d9faedbfbe9f7afbc1104809a04feb518db84f
Is my understanding right. I see this is the total images in this research. 

Any suggestions will really help. Thank you.

**Comment by FMCalisto** [comment, marked as answer]
Yes, you can follow that direction. Any question specific to that, just tell me.

**Reply by Kolaparthi-Vamsi** [reply]
Thank you Calisto. Hope you are doing great. I talked to my data science director yesterday and she is asking me to try to specifically focus on MMIC dataset. The idea remains the same. To identity the degree of cancer stage through image and auto generate a report. So for training now obviously I will  need images and text reports as well. I am doing hard work and my best to get the data and I request you if you can research and let me know if you find any relevant to my work.  My humble request Calisto.

**Reply by FMCalisto** [reply]
You can try these *datasets* that may help you on that:

- [Duke Breast Cancer MRI Dataset](https://sites.duke.edu/mazurowski/resources/breast-cancer-mri-dataset/)

- [Breast Cancer Wisconsin (Diagnostic) Data Set](https://www.kaggle.com/datasets/uciml/breast-cancer-wisconsin-data)

- [CBIS-DDSM: Breast Cancer Image Dataset](https://www.kaggle.com/datasets/awsaf49/cbis-ddsm-breast-cancer-image-dataset)
